pkg/util/errors/errors.go

```diff
@@ -21,6 +21,13 @@ import (
 	"net/http"
 )
 
+type StatusCode int
+
+const (
+	NotFoundStatusCode  StatusCode = 404
+	ForbiddenStatusCode StatusCode = 403
```

Comment:
You can just use the constants from `net/http`, no need to define your own set of HTTP status code constants.